python/ray/dashboard/client/src/App.tsx

```diff
@@ -91,6 +95,12 @@ export type GlobalContextType = {
    * The name of the current selected datasource.
    */
   dashboardDatasource: string | undefined;
+  /**
+   * The name of the current server-side timezone.
+   */
+  serverTimeZone: string | undefined;
+
+  serverTimeZoneLoaded: boolean | undefined;
```

Comment:
let's combine these into a single variable.

`serverTimeZone` will either be undefined if it's unloaded. null, if we couldn't fetch it, or the value from the server.